src/functions/products/getProductsById/getProductsById.ts

```diff
@@ -0,0 +1,33 @@
+import { ProductType } from '@functions/products/types';
+import { ProductService } from '@services/products/ProductService';
+import {
+  createAPIGatewayProxyHandler,
+  formatJSONResponse,
+} from '@libs/api-gateway';
+import { ErrorMessage } from '@constants/errors';
+import { PathParams } from '@typings/APIGateway.types';
+
+export const getProductsById = createAPIGatewayProxyHandler<
+  PathParams<Pick<ProductType, 'id'>>
+>(async event => {
+  const paramId = event.pathParameters.id;
+  const productsService = new ProductService();
+
+  try {
+    const product = await productsService.findById(paramId);
+
+    if (!product) {
+      return formatJSONResponse({
+        message: ErrorMessage.PRODUCT_NOT_FOUND,
+      });
+    }
+
+    return formatJSONResponse(product, 200);
```

Comment:
I wasn't able to get a success response. Not sure why `productId` is required, and how to make that request properly.

<img width="902" alt="Screenshot 2023-03-02 at 13 30 44" src="https://user-images.githubusercontent.com/8228505/222388976-6d0756f8-310a-4703-8d3a-3e01203bf5c5.png">

Reply:
I just fixed the name for the param, should work now.